pysrc/Control_EnvironmentGUI.py

```diff
@@ -311,11 +309,14 @@ def create_message(self, message_type, data=None):
             formatted = {message_type: new_data}
             return json.dumps(formatted)
         elif message_type == "control":
-            new_data = {"playing": str(Global_access.running),
-                        "fps": str(Global_access.fps),
-                        "updateDisplay": str(Global_access.updateDisplay)}
-            formatted = {message_type: new_data}
-            return json.dumps(formatted)
+            return json.dumps({
+                "type": "control",
+                "control": {
+                    "playing": Global_access.running,
```

Comment:
i expect these to be boolean and ints, not converted to string

also new format, which all the rest of the branches in this if chain need to be converted to